config/initializers/mime_types.rb

```diff
@@ -2,3 +2,5 @@
 
 # Add new mime types for use in respond_to blocks:
 # Mime::Type.register "text/richtext", :rtf
+
+Mime::Type.register "application/graphql", :graphql
```

Comment:
I don't think you need this anymore, if you're not using ActionView to load the queries.

Reply:
Oops.  I'll remove it.